webcompat/static/js/lib/issues.js

```diff
@@ -489,6 +490,23 @@ issues.MainView = Backbone.View.extend({
 
   handleKeyShortcuts: function() {
     Mousetrap.bind("mod+enter", _.bind(this.addNewComment, this));
+  },
+
+  // Auto expanding feature for comments
```

Comment:
In general, duplicating code is not great... but I think we have this problem in a few other places. Something that #909 (or similar) should be able to fix. Can we at least leave a comment pointing to the other dupe'd code so they stay in sync?